Gemfile.lock

```diff
@@ -523,14 +523,16 @@ GEM
     thor (1.1.0)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    traject (2.3.4)
+    traject (3.5.0)
       concurrent-ruby (>= 0.8.0)
       dot-properties (>= 0.1.1)
-      hashie (~> 3.1)
+      hashie (>= 3.1, < 5)
+      http (>= 3.0, < 5)
```

Comment:
I'll try to loosening this pinning in https://github.com/traject/traject/pull/270.